data/distro.yml

```diff
@@ -635,3 +635,60 @@
         <p>Note: for now, graphical installation of Flatpak apps is not possible on Pisi GNU/Linux.</p>
       </li>
     </ol>
+
+- name: "KDE Neon"
+  logo: "kde-neon.svg"
+  info: >
+    <h1>KDE Neon 5.19 and later has Flatpak installed by default.</h1>
+    <p>To activate Flatpak's built-in support just go to Discover > Settings > Flatpak > Add Flathub and mark Flatpak as Default.</p>
+    <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+    <p></p>
+    <p>In previous versions you must add the URI of the repository to enable Flathub:</p>
+    <ol>
+      <li>
+        <p>Download the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a>.</p>
+      </li>
+      <li>
+        <p>Go to Discover > Sources > Add Source and select the repository file.</p>
+      </li>
+    <ol>
+
+- name: "OpenMandriva"
+  logo: "openmandriva-blu.svg"
+  info: >
+    <h1>Flatpak and the Discover backend can be installed from Dnfdragora (software manager) starting from OpenMandravia LX 4.</h1>
```

Comment:
We should specify how to do this

Reply:
it should say "OpenMandriva" not OpenMandravia

Reply:
@mwleeds we should also add the command line instructions: 

Install Flatpak
To install Flatpak on OpenMandriva Lx4, run the following in a terminal:
$ sudo dnf install flatpak

Add the Flathub repository
Flathub is the best place to get Flatpak apps. To enable it, run:
$ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

Restart
To complete setup, restart your system. Now all you have to do is install some apps